src/libsync/account.cpp

```diff
@@ -40,10 +41,12 @@ Q_LOGGING_CATEGORY(lcAccount, "sync.account", QtInfoMsg)
 
 Account::Account(QObject *parent)
     : QObject(parent)
+    , _uuid(QUuid::createUuid())
```

Comment:
Why is a new Uuid created whenever an account object is constructed?

Reply:
afaict `acc->_uuid = settings.value(userUUIDC(), acc->_uuid).toUuid();` this is where it's overridden anyway. 
Maybe we should leave it empty/invalid and fallback to QUuid::createUuid() where it's overridden

Reply:
But a newly created account object that is not loaded from settings should also have one, thats why I generate one.
I could also make it lazy and generate it in the getter but...